src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,93 @@
+package org.example;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        while (true) {
+            System.out.print("\n Выберите задание: ");
+            int smth = in.nextInt();
+            switch (smth) {
+                case 1:
+                    task1();
+                    break;
+                case 2:
+                    task2();
+                    break;
+                case 3:
+                    task3();
+                    break;
+                case 4:
+                    task4();
+                    break;
+                case 5:
+                    task5();
+                    break;
+            }
+        }
+    }
+
+    public static void task1() {
```

Comment:
```suggestion
    public static void getTask1() {
```
приучайся верно именовать методы, по джава конвенции